### PR TITLE
get history from myft-client and not long abandonned user-prefs route

### PR DESCRIPTION
**Pull request by laurieboyes**
Previously was trying to use `/__user-prefs/history` route, which has been broken since ???

Now using myft-client.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,45 +1,23 @@
...
-			return response.json();
-		});
-}
+const taxonomyBlacklist = ['organisations', 'topics', 'people', 'authors', 'brand', 'specialReports'];
 
 function setContent(el, topics) {
 	const topicForms = el.querySelectorAll('.n-myft-ui--follow');
 
-	[].slice.call(topicForms).forEach((form, index) => {
+	Array.from(topicForms).forEach((form, index) => {
```
Comment (keirog) [inline main.js]: Can we use this in front-end+babel land now? It never used to compile it with our setup

Comment (laurieboyes) [inline main.js]: Oh really? It's polyfilled

Comment (keirog) [inline main.js]: Ah yep, quite right. Maybe it wasn't polyfilled by default before?

File: main.js [hunk 2]
```diff
@@ -1,45 +1,23 @@
...
-
-	return fetch(url, options)
-		.then(response => {
-			if (response.status >= 400 && response.status < 600) {
-				throw new Error(`Bad response from server for ${url}`);
-			}
-			return response.json();
-		});
-}
+const taxonomyBlacklist = ['organisations', 'topics', 'people', 'authors', 'brand', 'specialReports'];
```
Comment (keirog) [inline main.js]: Isn't this a whitelist?

**Comment by keirog** [comment]
👍 👍

File: main.js (hunk 1 above)
Comment (laurieboyes) [inline main.js]: Ah yeah they did just make a load more default didn't they

File: main.js (hunk 2 above)
Comment (laurieboyes) [inline main.js]: Huh, uh, yeah

**Pull request merged**